docs/how-to/how-to-build-and-test-ccv0.md

```diff
@@ -176,7 +176,6 @@ there.
         ```
         total 72
         -rw-r--r--  1 root root  2977 Jan 20 10:03 config.json
-        -rw-r--r--  1 root root   372 Jan 20 10:03 umoci.json
         -rw-r--r--  1 root root 63584 Jan 20 10:03 sha256_be9faa75035c20288cde7d2cdeb6cd1f5f4dbcd845d3f86f7feab61c4eff9eb5.mtree
```

Comment:
mtree file is also created by `umoci`, with `image-rs` we only have `config.json` and `rootfs`

Reply:
Thanks, removed